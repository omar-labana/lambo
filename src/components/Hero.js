import pizza from '../assets/pizza.png';

const Hero = () => (
  <div className="flex justify-around items-center py-32 mx-32">
    <div className="text-gray-300 text-3xl w-96">World's most delicious food you'll want to try!</div>
    <img src={pizza} alt="Hero pizza" className="w-96" />
  </div>
);

export default Hero;
