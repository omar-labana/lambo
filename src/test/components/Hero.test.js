import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Hero from '../../components/Hero';


describe('Hero Component', () => {
  const wrapper = shallow(<Hero />);

  it('exists', () => {
    expect(wrapper.find('Hero')).toBeTruthy();
  });

  it('has to match snap', () => {
    const home = renderer.create(<Hero />);
    expect(home).toMatchSnapshot();
  });
});