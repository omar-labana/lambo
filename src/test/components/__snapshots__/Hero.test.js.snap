// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Component has to match snap 1`] = `
<div
  className="flex justify-around items-center py-32 mx-32"
>
  <div
    className="text-gray-300 text-3xl w-96"
  >
    World's most delicious food you'll want to try!
  </div>
  <img
    alt="Hero pizza"
    className="w-96"
    src="pizza.png"
  />
</div>
`;
